pkg/controller/operators/olm/operator.go

```diff
@@ -950,36 +951,42 @@ func (a *Operator) handleClusterServiceVersionDeletion(obj interface{}) {
 		logger.WithError(err).Warn("cannot list cluster role bindings")
 	}
 	for _, crb := range crbs {
-		syncError := a.objGCQueueSet.RequeueEvent("", kubestate.NewResourceEvent(kubestate.ResourceUpdated, crb))
-		logger.Debugf("handleCSVdeletion - requeued update event for %v, res=%v", crb, syncError)
+		if err := a.objGCQueueSet.RequeueEvent("", kubestate.NewResourceEvent(kubestate.ResourceUpdated, crb)); err != nil {
+			logger.WithError(err).Warnf("failed to requeue gc event: %v", crb)
+		}
 	}
 
 	crs, err := a.lister.RbacV1().ClusterRoleLister().List(ownerSelector)
 	if err != nil {
 		logger.WithError(err).Warn("cannot list cluster roles")
 	}
 	for _, cr := range crs {
-		syncError := a.objGCQueueSet.RequeueEvent("", kubestate.NewResourceEvent(kubestate.ResourceUpdated, cr))
-		logger.Debugf("handleCSVdeletion - requeued update event for %v, res=%v", cr, syncError)
+		if err := a.objGCQueueSet.RequeueEvent("", kubestate.NewResourceEvent(kubestate.ResourceUpdated, cr)); err != nil {
+			logger.WithError(err).Warnf("failed to requeue gc event: %v", cr)
+		}
 	}
 
 	webhookSelector := labels.SelectorFromSet(ownerutil.OwnerLabel(clusterServiceVersion, v1alpha1.ClusterServiceVersionKind)).String()
 	mWebhooks, err := a.opClient.KubernetesInterface().AdmissionregistrationV1().MutatingWebhookConfigurations().List(context.TODO(), metav1.ListOptions{LabelSelector: webhookSelector})
 	if err != nil {
 		logger.WithError(err).Warn("cannot list MutatingWebhookConfigurations")
 	}
-	for _, webhook := range mWebhooks.Items {
-		syncError := a.objGCQueueSet.RequeueEvent("", kubestate.NewResourceEvent(kubestate.ResourceUpdated, &webhook))
-		logger.Debugf("handleCSVdeletion - requeued update event for %v, res=%v", webhook, syncError)
+	for i := 0; i < len(mWebhooks.Items); i++ {
```

Comment:
you can still keep the `range`, just make sure to do `w := webhook` in the loop body